noir-projects/noir-protocol-circuits/crates/blob/src/blob.nr

```diff
@@ -517,79 +412,31 @@ mod tests {
         assert(expected_y == output.y);
     }
 
-    // All hardcoded values in this test are taken from yarn-project/foundation/src/blob/blob.test.ts -> 'should evaluate full blobs'
+    // All hardcoded values in this test are taken from yarn-project/foundation/src/blob/blob.test.ts -> 'should evaluate full blob'
     #[test]
-    unconstrained fn test_full_blobs() {
-        let mut blob: [Field; FIELDS_PER_BLOB * BLOBS_PER_BLOCK] =
-            [0; FIELDS_PER_BLOB * BLOBS_PER_BLOCK];
-        for j in 0..BLOBS_PER_BLOCK {
-            for i in 0..FIELDS_PER_BLOB {
-                blob[j * FIELDS_PER_BLOB + i] = i as Field + 2;
-            }
+    unconstrained fn test_full_blob() {
+        let mut blob: [Field; FIELDS_PER_BLOB] = [0; FIELDS_PER_BLOB];
+        for i in 0..FIELDS_PER_BLOB {
+            blob[i] = i as Field + 2;
         }
+        let mut sponge_blob = SpongeBlob::new(FIELDS_PER_BLOB);
+        sponge_blob.absorb(blob, FIELDS_PER_BLOB);
 
-        let mut sponge_blob = SpongeBlob::new(FIELDS_PER_BLOB * BLOBS_PER_BLOCK);
-        sponge_blob.absorb(blob, FIELDS_PER_BLOB * BLOBS_PER_BLOCK);
-
-        let kzg_commitment_in = BlobCommitment {
-            inner: [
-                0x00ac771dea41e29fc2b7016c32731602c0812548ba0f491864a4e03fdb94b8d3,
-                0x000000000000000000000000000000d195faad1967cdf005acf73088b0e8474a,
-            ],
-        };
+        let kzg_commitment_in = [
+            0x00ac771dea41e29fc2b7016c32731602c0812548ba0f491864a4e03fdb94b8d3,
+            0x000000000000000000000000000000d195faad1967cdf005acf73088b0e8474a,
+        ];
 
-        let output = evaluate_blobs(blob, [kzg_commitment_in; BLOBS_PER_BLOCK], sponge_blob);
+        let hashed_blob = sponge_blob.squeeze();
+        let output = evaluate_blob(blob, kzg_commitment_in, hashed_blob);
 
-        // y is a BLS field with value 0x52fd4e272015a79f3889cc9ab1d84bee4326de7d8ced52612ecc9ec137bd38ee
+        // y is a BLS field with value 0x0365 494e66a289c4509ecf97af4ff92aa7 ecc38f478ced014b6ae860502a1b1c
         let expected_y: F = F::from_limbs([
-            0x26de7d8ced52612ecc9ec137bd38ee,
-            0x4e272015a79f3889cc9ab1d84bee43,
-            0x52fd,
+            0xecc38f478ced014b6ae860502a1b1c,
+            0x494e66a289c4509ecf97af4ff92aa7,
+            0x0365,
         ]);
```

Comment:
Modified this test to evaluate one full blob, since we have plenty of tests which evaluate multiple blobs and batch them in `blob_batching.nr`. This test singles out barycentric evaluation of a blob with all fields filled.